pkg/controller/migplan/validation.go

```diff
@@ -37,10 +39,13 @@ const (
 	InvalidDestinationCluster                  = "InvalidDestinationCluster"
 	NsNotFoundOnSourceCluster                  = "NamespaceNotFoundOnSourceCluster"
 	NsNotFoundOnDestinationCluster             = "NamespaceNotFoundOnDestinationCluster"
+	InvalidNamespaceAuthorization              = "InvalidNamespaceAuthorization"
```

Comment:
The `Invalid` seem a bit odd.  The condition is that the identity has insufficient privileges to operation on 1+ of the namespaces.  This working suggests the authorization itself is invalid.  Perhaps consider: `UnauthorizedNamespaces`.